example/lib/main.dart

```diff
@@ -265,47 +265,75 @@ class _MyAppState extends State<MyApp> {
       'gender': 'User Gender',
       'byear': '1989',
     };
-    Countly.setUserData(options);
+    Countly.instance.userProfile.setUserProperties(options);
+  }
+
+  void setProperties() {
+    Map<String, Object> userProperties = {
+      'name': 'Name of User',
+      'username': 'Username',
+      'email': 'User Email',
+      'organization': 'User Organization',
+      'phone': 123456789,
+      'picture': 'https://count.ly/images/logos/countly-logo.png',
+      'picturePath': '',
+      'gender': 'User Gender',
+      'byear': '1989',
+      'Custom Integer': 123,
+      'Custom String': "Some String",
+      'Custom Array': ['array value 1', 'array value 2'],
+      'Custom Map': {'key 1' : 'value 1', 'key 2' : 'value 2'},
+    };
+
+    Countly.instance.userProfile.setUserProperties(userProperties);
   }
 
   void setProperty() {
-    Countly.setProperty('setProperty', 'My Property');
+    Countly.instance.userProfile.setProperty('setProperty', 'My Property');
   }
 
   void increment() {
-    Countly.increment('increment');
+    Countly.instance.userProfile.increment('increment');
   }
 
   void incrementBy() {
-    Countly.incrementBy('incrementBy', 10);
+    Countly.instance.userProfile.incrementBy('incrementBy', 10);
   }
 
   void multiply() {
-    Countly.multiply('multiply', 20);
+    Countly.instance.userProfile.multiply('multiply', 20);
   }
 
   void saveMax() {
-    Countly.saveMax('saveMax', 100);
+    Countly.instance.userProfile.saveMax('saveMax', 100);
   }
 
   void saveMin() {
-    Countly.saveMin('saveMin', 50);
+    Countly.instance.userProfile.saveMin('saveMin', 50);
   }
 
   void setOnce() {
-    Countly.setOnce('setOnce', '200');
+    Countly.instance.userProfile.setOnce('setOnce', '200');
   }
 
   void pushUniqueValue() {
-    Countly.pushUniqueValue('pushUniqueValue', 'morning');
+    Countly.instance.userProfile.pushUnique('pushUniqueValue', 'morning');
```

Comment:
name change?

Reply:
that would be inline with the android names